### PR TITLE
Style: "Fix XCTAssertEqual Message"

### PR DESCRIPTION
**Pull request by wi-seong-cheol**
The value of the message in XCTAssertEqual has been modified because the value of the message is invalid.

### CODE REVIEW

**Comment by dianaafanador3** [comment]
Please fix code factor errors

File: Realm/Tests/Swift/SwiftSetTests.swift [hunk 1]
```diff
@@ -150,59 +150,84 @@ class SwiftRLMSetTests: RLMTestCase {
...
+        min = noArray.min(ofProperty: "doubleCol") as! NSNumber
+        XCTAssertEqual(min.doubleValue, Double(2.5), accuracy: 0.1, "Minimum should be 2.5")
+        min = yesArray.min(ofProperty: "doubleCol") as! NSNumber
+        XCTAssertEqual(min.doubleValue, Double(0.0), accuracy: 0.1, "Minimum should be 0.0")
 
         // Test date min
-        let dateMinOutput = aggSet.set.min(ofProperty: "dateCol") as! Date
-        XCTAssertEqual(dateMinOutput, dateMinInput, "Minimum should be dateMaxInput")
+        var dateMinOutput = noArray.min(ofProperty: "dateCol") as! Date
+        XCTAssertEqual(dateMinOutput, dateMaxInput, "Minimum should be dateMaxInput")
```
Comment (dianaafanador3) [inline Realm/Tests/Swift/SwiftSetTests.swift:203]: For the assert message this could be  "Minimum should be \(dateMaxInput)", for all the date asserts

Comment (wi-seong-cheol) [reply Realm/Tests/Swift/SwiftSetTests.swift:203]: Same as line 363 in 'SwiftArrayTests' file. Then is the code in the SwiftArrayTests file also wrong?

Comment (dianaafanador3) [reply Realm/Tests/Swift/SwiftSetTests.swift:203]: I guess there is no need for this, it was just an extra

Comment (wi-seong-cheol) [reply Realm/Tests/Swift/SwiftSetTests.swift:203]: Then, should I change the code of 'SwiftSetTests' file to the previous one?

Comment (dianaafanador3) [reply Realm/Tests/Swift/SwiftSetTests.swift:203]: Nope don't worry.

**Pull request merged**